numpy/meson.build

```diff
@@ -143,12 +145,45 @@ foreach subdir: pure_subdirs
   install_subdir(subdir, install_dir: np_dir)
 endforeach
 
-custom_target('__config__.py',
-  output: '__config__.py',
+cc = meson.get_compiler('c')
+cpp = meson.get_compiler('cpp')
```

Comment:
These two lines aren't needed, the `cc` and `cpp` variables are already defined in the top-level `meson.build`

Reply:
refined on [766dcf4](https://github.com/numpy/numpy/pull/22769/commits/766dcf44ed586b6e738eaf86f963ab2740ea4c37)